gammapy/spectrum/tests/test_models.py

```diff
@@ -94,6 +95,19 @@ def table_model():
         integral_1_10TeV=u.Quantity(2.6034046173089, "cm-2 s-1"),
         eflux_1_10TeV=u.Quantity(5.340285560055799, "TeV cm-2 s-1"),
     ),
+    dict(
+        name="plsec_4fgl",
+        model=PLSuperExpCutoff4FGL(
+            index_1=1.5 * u.Unit(""),
```

Comment:
You can remove the `* u.Unit("")` here and in the next line, just write this:
```
index_1=1.5,
index_2=2,
```

Reply:
Done!